jsonschema-validation.xml

```diff
@@ -1017,13 +1017,16 @@
                 <t>
                     If the instance is a string, and if "contentMediaType" is present, this
                     property contains a schema which describes the structure of the string.
+                    The string MUST be in a format compatible with the data model defined in the
+                    <xref target="json-schema">core JSON Schema.</xref>
```

Comment:
I stared at this for a minute wondering what this was trying to say. Perhaps this addition would help? `"; that is, a valid JSON string."`

Reply:
That’s what I originally thought at first too. That’s incorrect, however. I refer you to the previous resolved comments in this PR.

Reply:
See: https://github.com/json-schema-org/json-schema-spec/pull/917#discussion_r428091385
(You even replied to it.)

Reply:
ok right.